collection_requirements.rst

```diff
@@ -1,73 +1,77 @@
-*****************************
-Ansible Collections Checklist
-*****************************
+**************************************************
+Ansible Community Package Collections Requirements
+**************************************************
 
 .. contents:: Topics
 
 Overview
 ========
-This document is for maintainers of collections to provide them help, advice, and guidance on making sure their collections are correct.
 
-**Warning:** Subject to frequent updates. This is a "living document", expect it to change as we progress with the Collections work over the next few months.
+This document is for maintainers of Ansible Community collections, living in the `ansible-collections <https://github.com/ansible-collections>`_ repository or included in the Ansible community package (hereinafter the Collections), to provide them help, advice, and guidance on making sure their collections are correct.
```

Comment:
```suggestion
This document is for maintainers of Ansible Community collections, living in the `ansible-collections <https://github.com/ansible-collections>`_ repository or included in the Ansible community package (hereinafter the collections), to provide them help, advice, and guidance on making sure their collections are correct.
```
Do we really have to use a capital C here? If we do, there are a LOT of more places to check and possibly adjust (I saw a few which would need a capital C as well). I think it's better to stick to the lower-case form.

Reply:
yep, let's use it in lower-case

Reply:
changed everywhere, thanks!